### ISSUE TITLE
Using google drive_sync() to put notebooks into /reports/ directory

### ISSUE THREAD

**Issue by ddkapan** [calacademy-research/COMB#55]
Matt's notebook is now in the following folder on our Google drive repository:
Resilience_data_drive > COMB > reports
https://drive.google.com/file/d/1RBuqlhmVAmrBuwPZ9ke7VJykKnkLia-3/view?usp=sharing
Since this blends data and code with exposition we will add that folder to .gitignore and build a synchronizer by running the script: 
drive_sync_reports.R from the command line.
Now a new 'feature' issue on the repo...

**Comment by matth79** [comment]
This should probably include adding a line to .gitignore

```
reports/
```

**Comment by ddkapan** [comment]
Agreed, let me test your overall improvements and interaction with the reports directory with the your last pull request on this topic.

**Comment by ddkapan** [comment]
See also /papers/ subdirectory ... forgot about this /reports/ suggestion and don't want to violate any GitHub development standards that we set unless they are not that big of a deal ... e.g. it isn't hard to sync a report since they are not to big...

**Comment by ddkapan** [comment]
Can discuss after main stuff working...
